src/Cake.Common/Tools/DNVM/Use/DNVMSettings.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Cake.Core.Tooling;
+
+namespace Cake.Common.Tools.DNVM.Use
+{
+    /// <summary>
+    /// Settings for the dnvm command
+    /// </summary>
+    public class DNVMSettings : ToolSettings
+    {
+        /// <summary>
+        /// Gets or sets the runtime to be used
+        /// </summary>
+        public string Runtime { get; set; }

```

Comment:
Checking DNVM [source](https://github.com/aspnet/dnvm/blob/dev/src/dnvm.ps1#L1123) there's only a few valid values so this should be an nullable DNVM Runtime enum.
`[ValidateSet("", "clr", "coreclr", "mono")]`